src/nls/root/menu.js

```diff
@@ -121,6 +121,21 @@ define(function (require, exports, module) {
             },
             CREATE_CLIPPING_MASK: "Create Clipping Mask",
             CREATE_ARTBOARD: "Create New Artboard",
+            NEW_ARTBOARD_FROM_TEMPLATE: {
```

Comment:
I think we need to find a way to _not_ duplicate all this information into this strings file, since it already exists elsewhere.